src/services/completions.ts

```diff
@@ -338,7 +338,12 @@ namespace ts.Completions {
     ): CompletionEntry | undefined {
         let insertText: string | undefined;
         let replacementSpan: TextSpan | undefined;
+
         const insertQuestionDot = origin && originIsNullableMember(origin);
+        if (insertQuestionDot && preferences.includeAutomaticOptionalChainCompletions === false) {

```

Comment:
This condition should be in `addSymbolOriginInfo` where we add symbols to the origin map in the first place.

Reply:
I mean in addPropertySymbol

Reply:
Oops, I guess I should clarify that I signed off that this change seems sensible for VS and not that the implementation looks correct.